pipeline/price_modelling/modeler_lstm_k_step.py

```diff
@@ -0,0 +1,505 @@
+import os
```

Comment:
This file is huge.. any chance it could be split in subroutines?

Reply:
The first 120 lines are data processing which should have been in a separate data processing module. 

I have not done so for two reasons, 1. I was in a hurry to get the model working, so I didn't want to work on 2 separate module. 2. The whole data processing module may change since phase two requires a complete different pipeline, and I didn't have a good idea how I want to restructure it.

Eventually, we will definitely restructure the input pipeline for the RNN and remove the chunk.

Reply:
I figured it was due to practical concerns, but maybe we can track it with a TODO..